### DISCUSSION TITLE
Initial diagnostics coming back from RA are not processed.

### DISCUSSION THREAD

**Discussion by chanced** [simrat39/rust-tools.nvim#294] [General]
Rust Analyzer runs `cargo check` after load. Those diagnostics are not being parsed by neovim. In order to get workspace wide diagnostics, at least one save has to occur.

This pertains to both rust-tools and lspconfig, which is why I'm not creating an issue for it. I assume rust-tools is either wrapping lspconfig or standing in place of it so I'm not sure where the appropriate place to file this as an issue.

I'm also not certain if its my config just yet. I plan to rip everything out and create a bare-bones setup to check. Sorry I haven't done that already but I wanted to post this here before I forgot.

**Comment by simrat39** [comment]
Yep, looks like a neovim/lspconfig issue, you should report it there as we don't do anything for diagnostics
